docs/030-qs_intro.md

```diff
@@ -22,15 +22,19 @@ OVOS provides a couple of different images specifically for the Raspberry Pi: a
 
 #### Raspberry Pi OS Latest Images
 
-OVOS provides a "Headless" image that is similar to the original `picroft` software from `MycroftAI`. It runs without a screen, is built on top of Raspberry Pi OS, and works with a RPi3b/b+. It is often referred to as **RaspOVOS**.
+OVOS provides "headless" images that is similar to the original `picroft` software from `MycroftAI`. It runs without a screen, is built on top of Raspberry Pi OS, and works with a RPi3b/b+.
 
-- [headless images](https://ovosimages.ziggyai.online/raspbian/development)
+OVOS also provides a couple of full functioning images with a GUI.
+
+- [raspOVOS images](https://ovosimages.ziggyai.online/raspbian/newest)
+  - These are **unstable** builds, and not always working.
+  - Backup images of previous builds are [located here](https://ovosimages.ziggyai.online/raspbian/development)
 
 [Instructions on flashing the image can be found here](#flashing_an_image.md).
 
 #### Buildroot Latest Image
 
-The most advanced and featureful is the Buildroot image. If you want the full GUI, this is currently your only choice. _Please note that this image is older and running on OVOS 0.0.6. A new Buildroot image is in development but not yet available._
+The most advanced and featureful is the Buildroot image. It is specifically built for embeded hardware such as the RPi.  **New hardware is being added**
```

Comment:
It's definitely the most *zoom look at 'er go* and possibly the most *wow!*, but I don't know if it's still fair to say that it's more featureful than its siblings.